File: app/src/main/java/com/appzonegroup/app/fasttrack/model/BVNRequest.java
package com.appzonegroup.app.fasttrack.model;

/**
 * Created by Joseph on 6/3/2016.
 */
public class BVNRequest {

    private long ID;
    private String BVN ;
    private String CustomerPhoneNumber ;
    private String CustomerAccountNumber ;
    private String CustomerPIN ;
    private String InstitutionCode ;
    private String IsSync;
    private String Remark;
    private String IsConfirmed;
    private String GeoLocation;

    private String AgentPhoneNumber;
    private String AgentPin;
    private String AdditionalInformation;

    public String getBVN() {
        return BVN;
    }

    public void setBVN(String BVN) {
        this.BVN = BVN;
    }

    public String getCustomerPhoneNumber() {
        return CustomerPhoneNumber;
    }

    public void setCustomerPhoneNumber(String customerPhoneNumber) {
       CustomerPhoneNumber = customerPhoneNumber;
    }

    public String getCustomerAccountNumber() {
        return CustomerAccountNumber;
    }

    public void setCustomerAccountNumber(String customerAccountNumber) {
        CustomerAccountNumber = customerAccountNumber;
    }

    public String getCustomerPIN() {
        return CustomerPIN;
    }

    public void setCustomerPIN(String customerPIN) {
        CustomerPIN = customerPIN;
    }

    public String getInstitutionCode() {
        return InstitutionCode;
    }

    public void setInstitutionCode(String institutionCode) {
        InstitutionCode = institutionCode;
    }

    public String getIsSync() {
        return IsSync;
    }

    public void setIsSync(String isSync) {
        IsSync = isSync;
    }

    public String getRemark() {
        return Remark;
    }

    public void setRemark(String remark) {
        Remark = remark;
    }

    public String getIsConfirmed() {
        return IsConfirmed;
    }

    public void setIsConfirmed(String isConfirmed) {
        IsConfirmed = isConfirmed;
    }

    public long getID() {
        return ID;
    }

    public void setID(long ID) {
        this.ID = ID;
    }

    public String getGeoLocation() {
        return GeoLocation;
    }

    public void setGeoLocation(String geoLocation) {
        GeoLocation = geoLocation;
    }

    public String getAgentPhoneNumber() {
        return AgentPhoneNumber;
    }

    public void setAgentPhoneNumber(String agentPhoneNumber) {
        AgentPhoneNumber = agentPhoneNumber;
    }

    public String getAgentPin() {
        return AgentPin;
    }

    public void setAgentPin(String agentPin) {
        AgentPin = agentPin;
    }

    public String getAdditionalInformation() {
        return AdditionalInformation;
    }

    public void setAdditionalInformation(String additionalInformation) {
        AdditionalInformation = additionalInformation;
    }
}

File: core/src/main/java/com/cluster/core/data/request/BVNRequest.kt
package com.cluster.core.data.request

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

/**
 * Created by Joseph on 6/3/2016.
 */

@Serializable
class BVNRequest {

    @SerialName("ID")
    var id: Long = 0

    @SerialName("BVN")
    var bvn: String? = null

    @SerialName("CustomerPhoneNumber")
    var customerPhoneNumber: String? = null

    @SerialName("CustomerAccountNumber")
    var customerAccountNumber: String? = null

    @SerialName("CustomerPIN")
    var customerPIN: String? = null

    @SerialName("InstitutionCode")
    var institutionCode: String? = null

    @SerialName("IsSync")
    var isSync: String? = null

    @SerialName("Remark")
    var remark: String? = null

    @SerialName("IsConfirmed")
    var isConfirmed: String? = null

    @SerialName("GeoLocation")
    var geoLocation: String? = null

    @SerialName("AgentPhoneNumber")
    var agentPhoneNumber: String? = null

    @SerialName("AgentPin")
    var agentPin: String? = null

    @SerialName("AdditionalInformation")
    var additionalInformation: String? = null
}
